Rakefile

```diff
@@ -29,7 +28,9 @@ namespace :resque do
     }
   end
 
-  task :workers => :clean_working_job_ids # add job cleaning as a pre-requisite
+  # Add job cleaning as a pre-requisite
+  task work: :clean_working_job_ids
```

Comment:
I thought about doing it like this at first but decided against it, I don't remember why :facepalm: if you can't think of a reason, we can do it like this.